cfr-heartbeat.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "HEARTBEAT_TACTIC_2_75",
+    "template": "cfr_urlbar_chiclet",
+    "content": {
+      "layout": "chiclet_open_url",
+      "category": "cfrHeartbeat",
+      "bucket_id": "HEARTBEAT_TACTIC_2_75,",
+      "notification_text": "Help improve Firefox",
+      "active_color": "#595e91",
+      "action": {
+        "url": "https://qsurvey.mozilla.com/s3/goals-survey",
+        "where": "tabshifted"
+      }
+    },
+    "priority": 4,
+    "targeting": "locale in [\"en-US\",\"en-CA\",\"en-AU\",\"en-GB\",\"en-ZA\",\"en-NZ\"] && firefoxVersion >= 75 && [userId, \"HEARTBEAT_TACTIC_2_75\"]|stableSample(0.01)",
```

Comment:
Targeting requirements: https://docs.google.com/document/d/1t1v2gh4NJ6GHTh1DXvPq81cG18-2E-GFdWSxjhm7dtg/edit

Reply:
It is possible to encode the date as well but I'm not sure if we should do it.

Reply:
Is it possible to use `localeLanguageCode == "en"` here?

Reply:
Sure will update.